iOS/Flows/Conversation/Emotion Detail/EmotionDetailCollectionViewDataSource.swift

```diff
@@ -0,0 +1,41 @@
+//
+//  EmotionDetailCollectionViewDataSource.swift
+//  Jibber
+//
+//  Created by Martin Young on 4/25/22.
+//  Copyright © 2022 Benjamin Dodgson. All rights reserved.
+//
+
+import Foundation
+
+typealias EmotionDetailSection = EmotionDetailCollectionViewDataSource.SectionType
+typealias EmotionDetailItem = EmotionDetailCollectionViewDataSource.ItemType
+
+class EmotionDetailCollectionViewDataSource: CollectionViewDataSource<EmotionDetailSection,
+                                             EmotionDetailItem> {
```

Comment:
The layout and data source below are similar but simpler than the emotion selection screen.